### ISSUE TITLE
Return the number of atoms in the SDBlock

### ISSUE THREAD

**Issue by coleb** [pechersky/sendoff#18]
Would be valuable to return the number of atoms in the SDBlock for quick debugging of records. Note, this needs a different code path for V2000 vs V3000 records.

**Comment by pechersky** [comment]
@coleb Do you want to trust the `COUNTS` line or the actual number of atom lines?

**Comment by coleb** [comment]
In V2000 the counts line is very much required because there is no other demarcation of what an ATOM line is in the format short of some much more complex parsing logic around the format of a line. So I think it best to keep the behavior of V2000 and V3000 as equivalent as possible by just relying on the counts line in both circumstances. 

Was good for me to just check the specification though, and the line is a required line for V3000:

```
A counts line is required, and must be first. It specifies the number of atoms, bonds, 3D objects,
and Sgroups. It also specifies whether or not the CHIRAL flag is set. Optionally, the counts line
can specify molregno. This is only used when the regno exceeds 999999 (the limit of the format
in the molfile header line). The format of the counts line is:
M V30 COUNTS na nb nsg n3d chiral [REGNO=regno]
```

**Comment by pechersky** [comment]
Closed by #22.
